documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -180,8 +180,8 @@ include::modules/adding-source-provider.adoc[leveloffset=+4]
 
 include::modules/selecting-migration-network-for-virt-provider.adoc[leveloffset=+4]
 
-include::modules/creating-network-mapping.adoc[leveloffset=+2]
-include::modules/creating-storage-mapping.adoc[leveloffset=+2]
+// include::modules/creating-network-mapping.adoc[leveloffset=+2]
```

Comment:
@RichardHoch - i think simply commenting them out is the best solution.

If we had more time, I would use an `ifdef` statement using the `{project-z-version}` variable to reveal at on a future release but this is the quickest solution given the time constraints.